src/components/NewsLetter/NewsLetter.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Container, Col, Row, InputGroup, FormControl } from 'react-bootstrap';
+import ButtonImg from './images/Slice 1.png';
+import './style.css';
+function NewsLetter() {
+  return (
+    <Container className="NewsLetter">
```

Comment:
With `PascalCase`, the first letter of every word in the identifier is upper case. With `camelCase`, the first letter of the first word in the identifier is lower case.

we are using camelCase so make the first letter lower case